src/app/api/v1/github/getGithubMonthlyStats.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019-2025 @polkassembly/fellowship authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { FELLOWS_REPO } from '@/global/constants/fellowsRepo';
+
+export const getGithubMonthlyStats = async () => {
+	const currentDate = new Date();
+	const currentYear = currentDate.getFullYear();
+	const currentMonth = currentDate.getMonth() + 1; // Months are zero-based in JavaScript
+	const [currentMonthFirstDay, currentMonthLastDay] = [new Date(currentYear, currentMonth - 1, 1), new Date(currentYear, currentMonth, 0)];
+
+	const previousMonth = currentMonth - 1 === 0 ? 12 : currentMonth - 1;
+	const [previousMonthFirstDay, previousMonthLastDay] = [new Date(currentYear, previousMonth - 1, 1), new Date(currentYear, previousMonth, 0)];
+
+	const [currentMonthData, previousMonthData] = await Promise.all([
+		fetch(`https://api.github.com/repos/${FELLOWS_REPO}/commits?per_page=100&since=${currentMonthFirstDay.toISOString()}&until=${currentMonthLastDay.toISOString()}`).then((res) =>
```

Comment:
please see if any of these responses from github can be cached, we don't want to be hitting the rate limit